os/include/debug.h

```diff
@@ -1030,6 +1030,12 @@ int get_errno(void);
 #define videoivdbg(...)
 #endif
 
+#ifdef CONFIG_DEBUG_SYSTEM_FAULTS
```

Comment:
Sorry for mis-communication.
I meant that
1. keep current code
2. add one more in else statement too

In other words, you should add this definition both under `#ifdef CONFIG_CPP_HAVE_VARARGS` and under `else`.

Reply:
Done.